requirements/test.in

```diff
@@ -8,7 +8,7 @@ Sphinx==1.8.5
 twine==1.14.0
 pytest==4.6.5
 pytest-runner==5.1
-httpretty
+httpretty<1.1.0  # can remove constraint once https://github.com/gabrielfalcao/HTTPretty/issues/425 is fixed
```

Comment:
Calling out this change - this and the `base.in` change to bump prefect to 0.15.1 are the only manual changes made. Everything else was just `make upgrade`